common/frame.go

```diff
@@ -948,21 +948,19 @@ func (f *Frame) FrameElement() (*ElementHandle, error) {
 }
 
 // GetAttribute of the first element found that matches the selector.
-func (f *Frame) GetAttribute(selector, name string, opts goja.Value) any {
+func (f *Frame) GetAttribute(selector, name string, opts goja.Value) (any, error) {
```

Comment:
`GetAttribute` should only returns a `string`. Having taken a quick look at the call stack it looks like we're using [element.getAttribute](https://github.com/grafana/xk6-browser/blob/5711416a992a522b549917eaf9bfffd771a81679/common/element_handle.go#L314), maybe a `string` instead of `any`?

Reply:
How about returning a `bool` to detect that in the mapping layer?

```
func (f *Frame) GetAttribute(selector, name string, opts goja.Value) (string, bool, error)
```

Even so, this should better go to a new issue because it'd change the existing signatures. I've tried to keep them intact other than adding errors.